cdb2jdbc/src/main/java/com/bloomberg/comdb2/jdbc/Comdb2Handle.java

```diff
@@ -2116,7 +2130,9 @@ public void close() throws IOException {
         if (opened == true) {
             synchronized (lock) {
                 if (opened == true) {
-                    if (ack) ack();
+                    if (ack && !skipDrain) {
```

Comment:
This should be placed under its own tunable. Consider using "[clear_ack](https://github.com/bloomberg/comdb2/blob/master/cdb2api/cdb2api.h#L251)" to be in line with the C api.

Reply:
Do you mean not passing it as an argument on the JDBC URL, but set it as a tunable on the database itself?

That change will be broader than what I intended, as it will affect all connections, not just those that need this particular behavior.

Reply:
skip_rs_drain controls whether unread rows are drained on close; clear_ack controls whether events are consumed on close. Two different arguments.

Reply:
Added a new `clear_ack` flag as suggested.